src/templates/next/components/UnorderedList/UnorderedList.tsx

```diff
@@ -0,0 +1,25 @@
+import type { UnorderedListProps } from "~/common"
+import OrderedList from "../OrderedList"
+import Paragraph from "../Paragraph"
+
+const UnorderedList = ({ items }: UnorderedListProps) => {
+  return (
+    <ul className="list-disc ps-8 my-1">
```

Comment:
from eye balling it seems like padding in design is more generous
https://www.figma.com/file/8TEzSSDbx97gctRJ7nKdqG/Website-Template-Gen-2?type=design&node-id=1516-13636&mode=design&t=cDs1tIIi9QBLaFaE-0

Reply:
Hmm I feel might be a bit too generous but sure, updated in 7bc1d1a154830958de76e9cb89f531c5ddd2f78f.